test/fixtures/json/valid.jsonc

```diff
@@ -1,6 +1,7 @@
 {
 	"max-len":
-																									[]// Off: max-len
+																									[], // Off: max-len
+	"bidi": "‎LTR"// Off: security/detect-bidi-characters
```

Comment:
GitHub doesn't render it, but there is an 0x200E LTR marker character to the left of `LTR`